ci/utility/jobs/tag-uc-feature/dev-uc-feature-object-tagging.yml

```
@@ -0,0 +1,28 @@
+jobs:
+  - name: uc-feature-object-tagging-dev
+    max_in_flight: 1
+    serial: true
+    plan:
+      - get: rbac-csv
+        trigger: true
+        passed:
+          - rbac-csv-upload-dev
+      - put: meta
+        resource: meta-development
+      - get: aws-common-infrastructure
+      - get: dataworks-aws-s3-object-tagger
+      - .: (( inject meta.plan.terraform-bootstrap ))
+      - .: (( inject meta.plan.terraform-output-aws-s3-object-tagger ))
+        config:
+          params:
+            TF_WORKSPACE: default
+      - .: (( inject meta.plan.terraform-output-common ))
+        config:
+          params:
+            TF_WORKSPACE: default
+      - .: (( inject meta.plan.object-tagging-batch ))
+        config:
+          params:
+            AWS_ROLE_ARN: arn:aws:iam::((aws_account.development)):role/ci
+            OUTPUTS_PREFIX_NAME: uc-feature_object_tagger_data_classification
```

```suggestion
            OUTPUTS_PREFIX_NAME: uc_feature_object_tagger_data_classification
```

Each job has the same issue.